rules/windows/execution_via_compiled_html_file.toml

```diff
@@ -19,18 +19,18 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Process Activity via Compiled HTML File"
```

Comment:
this rule generic and does not provide any context to end user, better add process.args : "-decode" or suspicious hh.exe child process (cmd.exe, powershell.exe, cscript.exe, wscript.exe).

Reply:
ignore the above if deprecating a rule will be handled in a different release/PR.

Reply:
Thanks @Samirbous. I see a couple mentions of deprecations. I would open an issue for `deprecation review for 7.13` and include any recommended deprecations (within this PR or _any_ other rules) and justifications per rule, and then we can do a single deprecate PR

Reply:
changed the query to match the malicious usage context of chm files

![image](https://user-images.githubusercontent.com/64742097/114831315-875fef80-9dcd-11eb-887b-4e655cc7930b.png)